packages/authentication/src/auth-component.ts

```diff
@@ -14,8 +14,8 @@ export class AuthenticationComponent implements Component {
   // TODO(bajtos) inject configuration
   constructor() {
     this.providers = {
-      [AuthenticationBindings.AUTH_ACTION]: AuthenticationProvider,
-      [AuthenticationBindings.METADATA]: AuthMetadataProvider,
+      [AuthenticationBindings.AUTH_ACTION.toString()]: AuthenticationProvider,
```

Comment:
I find `.toString` to be messy. Any reason we can't just access the `key` property of the `BindingKey`?

`AuthenticationBindings.AUTH_ACTION.key`

Reply:
+1 to use `AuthenticationBindings.AUTH_ACTION.key`.